### PR TITLE
fix: hard code testimonials due to Twitter API changes

### LINKED ISSUE #1199: bug (web page): testimonials on the web page only show a single tweet thrice
### Provide environment information

```
  System:
    OS: Linux 6.1 NixOS 23.05 (Stoat) 23.05 (Stoat)
    CPU: (8) x64 11th Gen Intel(R) Core(TM) i5-1135G7 @ 2.40GHz
    Memory: 10.81 GB / 15.42 GB
    Container: Yes
    Shell: 3.6.0 - /run/current-system/sw/bin/fish
  Binaries:
    Node: 18.14.0 - /etc/profiles/per-user/tommy/bin/node
    npm: 9.3.1 - /etc/profiles/per-user/tommy/bin/npm
```

Just to be clear, this is regarding https://create.t3.gg, I've seen this after just opening the web page, but I also recreated it by forking and the repo and running the page locally.

### Describe the bug

The testimonials at the bottom of https://create.t3.gg only show the same tweet three times, instead of four different tweets as defined in [www/src/components/landingPage/tweets/tweets.astro line 54](https://github.com/t3-oss/create-t3-app/blob/005c440a65f08ada707af8f43aa46ae4c11e02fa/www/src/components/landingPage/tweets/tweets.astro#L54).

This does not seem to be rendering bug, because this is the html that's being generated (locally as well as when visiting the live page):

```html
  <div class="flex flex-1 flex-wrap justify-start">
    <div class="flex flex-1 items-center">
      <div>
        <h3 class="mr-2 font-bold hover:underline">
          <a href="https://twitter.com/ajcwebdev" target="_blank" rel="noopener noreferrer" class="text-sm hover:text-[#1da1f2] sm:text-base">
            Anthony (ajcwebdev.x)
          </a>
        </h3>
        
        <a href="https://twitter.com/ajcwebdev" target="_blank" rel="noopener noreferrer" class="mr-1 whitespace-nowrap text-sm text-t3-purple-100 hover:text-[#1da1f2] lg:text-sm">
          @ajcwebdev
        </a>
      </div>
    </div>
    <a href="https://twitter.com/ajcwebdev/status/1544909672137867264" target="_blank" rel="noopener noreferrer" class="whitespace-nowrap text-sm text-t3-purple-100 hover:text-[#1da1f2] lg:text-sm">
      Thu Jul 07 2022
    </a>
  </div>
  <div class="w-full">
    <p class="my-1 whitespace-pre-line text-sm xl:text-base">Now that Blitz.js has pivoted and Bison has stagnated, create-t3-app will be the only framework to give Redwood a run for its money in the quest to build a legitimate fullstack React framework.</p>

    <div class="flex items-center justify-start py-2">
      <div class="mr-8 flex items-center text-t3-purple-100 hover:text-blue-500">
        <a aria-label="Comment on Tweet" href="https://twitter.com/intent/tweet?in_reply_to=1544909672137867264" target="_blank" rel="noopener noreferrer" class="flex h-8 w-8 items-center justify-center rounded-full hover:text-blue-500">
          <svg viewBox="0 0 24 24" class="h-5 w-5 fill-current">
            <g>
              <path d="M14.046 2.242l-4.148-.01h-.002c-4.374 0-7.8 3.427-7.8 7.802 0 4.098 3.186 7.206 7.465 7.37v3.828c0 .108.044.286.12.403.142.225.384.347.632.347.138 0 .277-.038.402-.118.264-.168 6.473-4.14 8.088-5.506 1.902-1.61 3.04-3.97 3.043-6.312v-.017c-.006-4.367-3.43-7.787-7.8-7.788zm3.787 12.972c-1.134.96-4.862 3.405-6.772 4.643V16.67c0-.414-.335-.75-.75-.75h-.396c-3.66 0-6.318-2.476-6.318-5.886 0-3.534 2.768-6.302 6.3-6.302l4.147.01h.002c3.532 0 6.3 2.766 6.302 6.296-.003 1.91-.942 3.844-2.514 5.176z"></path>
            </g>
          </svg>
        </a>
        <span class="ml-1">9</span>
      </div>
      <div class="mr-8 flex items-center text-t3-purple-100 hover:text-green-500">
        <a aria-label="Retweet" href="https://twitter.com/intent/retweet?tweet_id=1544909672137867264" target="_blank" rel="noopener noreferrer" class="flex h-8 w-8 items-center justify-center rounded-full hover:text-green-500">
          <svg viewBox="0 0 24 24" class="h-5 w-5 fill-current">
            <g>
              <path d="M23.77 15.67c-.292-.293-.767-.293-1.06 0l-2.22 2.22V7.65c0-2.068-1.683-3.75-3.75-3.75h-5.85c-.414 0-.75.336-.75.75s.336.75.75.75h5.85c1.24 0 2.25 1.01 2.25 2.25v10.24l-2.22-2.22c-.293-.293-.768-.293-1.06 0s-.294.768 0 1.06l3.5 3.5c.145.147.337.22.53.22s.383-.072.53-.22l3.5-3.5c.294-.292.294-.767 0-1.06zm-10.66 3.28H7.26c-1.24 0-2.25-1.01-2.25-2.25V6.46l2.22 2.22c.148.147.34.22.532.22s.384-.073.53-.22c.293-.293.293-.768 0-1.06l-3.5-3.5c-.293-.294-.768-.294-1.06 0l-3.5 3.5c-.294.292-.294.767 0 1.06s.767.293 1.06 0l2.22-2.22V16.7c0 2.068 1.683 3.75 3.75 3.75h5.85c.414 0 .75-.336.75-.75s-.337-.75-.75-.75z"></path>
            </g>
          </svg>
        </a>
        <span class="ml-1">3</span>
      </div>
      <div class="mr-6 flex items-center text-t3-purple-100 hover:text-red-500">
        <a aria-label="Like" href="https://twitter.com/intent/like?tweet_id=1544909672137867264" target="_blank" rel="noopener noreferrer" class="flex h-8 w-8 items-center justify-center rounded-full hover:text-red-500">
          <svg viewBox="0 0 24 24" class="h-5 w-5 fill-current">
            <g>
              <path d="M12 21.638h-.014C9.403 21.59 1.95 14.856 1.95 8.478c0-3.064 2.525-5.754 5.403-5.754 2.29 0 3.83 1.58 4.646 2.73.814-1.148 2.354-2.73 4.645-2.73 2.88 0 5.404 2.69 5.404 5.755 0 6.376-7.454 13.11-10.037 13.157H12zM7.354 4.225c-2.08 0-3.903 1.988-3.903 4.255 0 5.74 7.034 11.596 8.55 11.658 1.518-.062 8.55-5.917 8.55-11.658 0-2.267-1.823-4.255-3.903-4.255-2.528 0-3.94 2.936-3.952 2.965-.23.562-1.156.562-1.387 0-.014-.03-1.425-2.965-3.954-2.965z"></path>
            </g>
          </svg>
        </a>
        <span class="ml-1">18</span>
      </div>
    </div>
  </div>
</article>
        </div><div class="w-full shrink-0 first:ml-0 last:mr-0">
          <article class="flex flex-wrap items-start rounded-lg border border-t3-purple-200/20 bg-white/10 bg-cover px-4 pt-4 pb-1 transition-colors hover:border-t3-purple-300/50 hover:bg-white/20 hover:no-underline">
  <img class="rounded-full w-12 h-12 mr-3 astro-UXNKDZ4E" alt="@ajcwebdev's Avatar" width="48" height="48" src="/assets/hK_Qr-Dx_normal_2tIPuV.webp" loading="lazy" decoding="async">


  <div class="flex flex-1 flex-wrap justify-start">
    <div class="flex flex-1 items-center">
      <div>
        <h3 class="mr-2 font-bold hover:underline">
          <a href="https://twitter.com/ajcwebdev" target="_blank" rel="noopener noreferrer" class="text-sm hover:text-[#1da1f2] sm:text-base">
            Anthony (ajcwebdev.x)
          </a>
        </h3>
        
        <a href="https://twitter.com/ajcwebdev" target="_blank" rel="noopener noreferrer" class="mr-1 whitespace-nowrap text-sm text-t3-purple-100 hover:text-[#1da1f2] lg:text-sm">
          @ajcwebdev
        </a>
      </div>
    </div>
    <a href="https://twitter.com/ajcwebdev/status/1544909672137867264" target="_blank" rel="noopener noreferrer" class="whitespace-nowrap text-sm text-t3-purple-100 hover:text-[#1da1f2] lg:text-sm">
      Thu Jul 07 2022
    </a>
  </div>
  <div class="w-full">
    <p class="my-1 whitespace-pre-line text-sm xl:text-base">Now that Blitz.js has pivoted and Bison has stagnated, create-t3-app will be the only framework to give Redwood a run for its money in the quest to build a legitimate fullstack React framework.</p>

    <div class="flex items-center justify-start py-2">
      <div class="mr-8 flex items-center text-t3-purple-100 hover:text-blue-500">
        <a aria-label="Comment on Tweet" href="https://twitter.com/intent/tweet?in_reply_to=1544909672137867264" target="_blank" rel="noopener noreferrer" class="flex h-8 w-8 items-center justify-center rounded-full hover:text-blue-500">
          <svg viewBox="0 0 24 24" class="h-5 w-5 fill-current">
            <g>
              <path d="M14.046 2.242l-4.148-.01h-.002c-4.374 0-7.8 3.427-7.8 7.802 0 4.098 3.186 7.206 7.465 7.37v3.828c0 .108.044.286.12.403.142.225.384.347.632.347.138 0 .277-.038.402-.118.264-.168 6.473-4.14 8.088-5.506 1.902-1.61 3.04-3.97 3.043-6.312v-.017c-.006-4.367-3.43-7.787-7.8-7.788zm3.787 12.972c-1.134.96-4.862 3.405-6.772 4.643V16.67c0-.414-.335-.75-.75-.75h-.396c-3.66 0-6.318-2.476-6.318-5.886 0-3.534 2.768-6.302 6.3-6.302l4.147.01h.002c3.532 0 6.3 2.766 6.302 6.296-.003 1.91-.942 3.844-2.514 5.176z"></path>
            </g>
          </svg>
        </a>
        <span class="ml-1">9</span>
      </div>
      <div class="mr-8 flex items-center text-t3-purple-100 hover:text-green-500">
        <a aria-label="Retweet" href="https://twitter.com/intent/retweet?tweet_id=1544909672137867264" target="_blank" rel="noopener noreferrer" class="flex h-8 w-8 items-center justify-center rounded-full hover:text-green-500">
          <svg viewBox="0 0 24 24" class="h-5 w-5 fill-current">
            <g>
              <path d="M23.77 15.67c-.292-.293-.767-.293-1.06 0l-2.22 2.22V7.65c0-2.068-1.683-3.75-3.75-3.75h-5.85c-.414 0-.75.336-.75.75s.336.75.75.75h5.85c1.24 0 2.25 1.01 2.25 2.25v10.24l-2.22-2.22c-.293-.293-.768-.293-1.06 0s-.294.768 0 1.06l3.5 3.5c.145.147.337.22.53.22s.383-.072.53-.22l3.5-3.5c.294-.292.294-.767 0-1.06zm-10.66 3.28H7.26c-1.24 0-2.25-1.01-2.25-2.25V6.46l2.22 2.22c.148.147.34.22.532.22s.384-.073.53-.22c.293-.293.293-.768 0-1.06l-3.5-3.5c-.293-.294-.768-.294-1.06 0l-3.5 3.5c-.294.292-.294.767 0 1.06s.767.293 1.06 0l2.22-2.22V16.7c0 2.068 1.683 3.75 3.75 3.75h5.85c.414 0 .75-.336.75-.75s-.337-.75-.75-.75z"></path>
            </g>
          </svg>
        </a>
        <span class="ml-1">3</span>
      </div>
      <div class="mr-6 flex items-center text-t3-purple-100 hover:text-red-500">
        <a aria-label="Like" href="https://twitter.com/intent/like?tweet_id=1544909672137867264" target="_blank" rel="noopener noreferrer" class="flex h-8 w-8 items-center justify-center rounded-full hover:text-red-500">
          <svg viewBox="0 0 24 24" class="h-5 w-5 fill-current">
            <g>
              <path d="M12 21.638h-.014C9.403 21.59 1.95 14.856 1.95 8.478c0-3.064 2.525-5.754 5.403-5.754 2.29 0 3.83 1.58 4.646 2.73.814-1.148 2.354-2.73 4.645-2.73 2.88 0 5.404 2.69 5.404 5.755 0 6.376-7.454 13.11-10.037 13.157H12zM7.354 4.225c-2.08 0-3.903 1.988-3.903 4.255 0 5.74 7.034 11.596 8.55 11.658 1.518-.062 8.55-5.917 8.55-11.658 0-2.267-1.823-4.255-3.903-4.255-2.528 0-3.94 2.936-3.952 2.965-.23.562-1.156.562-1.387 0-.014-.03-1.425-2.965-3.954-2.965z"></path>
            </g>
          </svg>
        </a>
        <span class="ml-1">18</span>
      </div>
    </div>
  </div>
</article>
        </div><div class="w-full shrink-0 first:ml-0 last:mr-0">
          <article class="flex flex-wrap items-start rounded-lg border border-t3-purple-200/20 bg-white/10 bg-cover px-4 pt-4 pb-1 transition-colors hover:border-t3-purple-300/50 hover:bg-white/20 hover:no-underline">
  <img class="rounded-full w-12 h-12 mr-3 astro-UXNKDZ4E" alt="@ajcwebdev's Avatar" width="48" height="48" src="/assets/hK_Qr-Dx_normal_2tIPuV.webp" loading="lazy" decoding="async">


  <div class="flex flex-1 flex-wrap justify-start">
    <div class="flex flex-1 items-center">
      <div>
        <h3 class="mr-2 font-bold hover:underline">
          <a href="https://twitter.com/ajcwebdev" target="_blank" rel="noopener noreferrer" class="text-sm hover:text-[#1da1f2] sm:text-base">
            Anthony (ajcwebdev.x)
          </a>
        </h3>
        
        <a href="https://twitter.com/ajcwebdev" target="_blank" rel="noopener noreferrer" class="mr-1 whitespace-nowrap text-sm text-t3-purple-100 hover:text-[#1da1f2] lg:text-sm">
          @ajcwebdev
        </a>
      </div>
    </div>
    <a href="https://twitter.com/ajcwebdev/status/1544909672137867264" target="_blank" rel="noopener noreferrer" class="whitespace-nowrap text-sm text-t3-purple-100 hover:text-[#1da1f2] lg:text-sm">
      Thu Jul 07 2022
    </a>
  </div>
  <div class="w-full">
    <p class="my-1 whitespace-pre-line text-sm xl:text-base">Now that Blitz.js has pivoted and Bison has stagnated, create-t3-app will be the only framework to give Redwood a run for its money in the quest to build a legitimate fullstack React framework.</p>

    <div class="flex items-center justify-start py-2">
      <div class="mr-8 flex items-center text-t3-purple-100 hover:text-blue-500">
        <a aria-label="Comment on Tweet" href="https://twitter.com/intent/tweet?in_reply_to=1544909672137867264" target="_blank" rel="noopener noreferrer" class="flex h-8 w-8 items-center justify-center rounded-full hover:text-blue-500">
          <svg viewBox="0 0 24 24" class="h-5 w-5 fill-current">
            <g>
              <path d="M14.046 2.242l-4.148-.01h-.002c-4.374 0-7.8 3.427-7.8 7.802 0 4.098 3.186 7.206 7.465 7.37v3.828c0 .108.044.286.12.403.142.225.384.347.632.347.138 0 .277-.038.402-.118.264-.168 6.473-4.14 8.088-5.506 1.902-1.61 3.04-3.97 3.043-6.312v-.017c-.006-4.367-3.43-7.787-7.8-7.788zm3.787 12.972c-1.134.96-4.862 3.405-6.772 4.643V16.67c0-.414-.335-.75-.75-.75h-.396c-3.66 0-6.318-2.476-6.318-5.886 0-3.534 2.768-6.302 6.3-6.302l4.147.01h.002c3.532 0 6.3 2.766 6.302 6.296-.003 1.91-.942 3.844-2.514 5.176z"></path>
            </g>
          </svg>
        </a>
        <span class="ml-1">9</span>
      </div>
      <div class="mr-8 flex items-center text-t3-purple-100 hover:text-green-500">
        <a aria-label="Retweet" href="https://twitter.com/intent/retweet?tweet_id=1544909672137867264" target="_blank" rel="noopener noreferrer" class="flex h-8 w-8 items-center justify-center rounded-full hover:text-green-500">
          <svg viewBox="0 0 24 24" class="h-5 w-5 fill-current">
            <g>
              <path d="M23.77 15.67c-.292-.293-.767-.293-1.06 0l-2.22 2.22V7.65c0-2.068-1.683-3.75-3.75-3.75h-5.85c-.414 0-.75.336-.75.75s.336.75.75.75h5.85c1.24 0 2.25 1.01 2.25 2.25v10.24l-2.22-2.22c-.293-.293-.768-.293-1.06 0s-.294.768 0 1.06l3.5 3.5c.145.147.337.22.53.22s.383-.072.53-.22l3.5-3.5c.294-.292.294-.767 0-1.06zm-10.66 3.28H7.26c-1.24 0-2.25-1.01-2.25-2.25V6.46l2.22 2.22c.148.147.34.22.532.22s.384-.073.53-.22c.293-.293.293-.768 0-1.06l-3.5-3.5c-.293-.294-.768-.294-1.06 0l-3.5 3.5c-.294.292-.294.767 0 1.06s.767.293 1.06 0l2.22-2.22V16.7c0 2.068 1.683 3.75 3.75 3.75h5.85c.414 0 .75-.336.75-.75s-.337-.75-.75-.75z"></path>
            </g>
          </svg>
        </a>
        <span class="ml-1">3</span>
      </div>
      <div class="mr-6 flex items-center text-t3-purple-100 hover:text-red-500">
        <a aria-label="Like" href="https://twitter.com/intent/like?tweet_id=1544909672137867264" target="_blank" rel="noopener noreferrer" class="flex h-8 w-8 items-center justify-center rounded-full hover:text-red-500">
          <svg viewBox="0 0 24 24" class="h-5 w-5 fill-current">
            <g>
              <path d="M12 21.638h-.014C9.403 21.59 1.95 14.856 1.95 8.478c0-3.064 2.525-5.754 5.403-5.754 2.29 0 3.83 1.58 4.646 2.73.814-1.148 2.354-2.73 4.645-2.73 2.88 0 5.404 2.69 5.404 5.755 0 6.376-7.454 13.11-10.037 13.157H12zM7.354 4.225c-2.08 0-3.903 1.988-3.903 4.255 0 5.74 7.034 11.596 8.55 11.658 1.518-.062 8.55-5.917 8.55-11.658 0-2.267-1.823-4.255-3.903-4.255-2.528 0-3.94 2.936-3.952 2.965-.23.562-1.156.562-1.387 0-.014-.03-1.425-2.965-3.954-2.965z"></path>
            </g>
          </svg>
        </a>
        <span class="ml-1">18</span>
      </div>
    </div>
  </div>
```

As you can see, all three big div blocks point at the same tweet.

### Reproduction repo

I guess this falls under the category "documentation issue"? The page is the landing page: https://create.t3.gg

### To reproduce

Either visit https://create.t3.gg, or, in a fresh clone on the `next` branch, run the dev server for the docs and navigate to it.

### Additional information

Some screenshots of what I'm talking about:

image

image

image

As you can see, not only does the slider only have three buttons, but the same tweet is shown in any one of those positions.

### PR DESCRIPTION
**Pull request by tbreslein**
Closes #1199 

## ✅ Checklist

- [x] I have followed every step in the [contributing guide](https://github.com/t3-oss/create-t3-app/blob/main/CONTRIBUTING.md) (updated 2022-10-06).
- [x] The PR title follows the convention we established [conventional-commit](https://www.conventionalcommits.org/en/v1.0.0/)
- [x] I performed a functional test on my final commit

---

## Changelog

Due to the new Twitter API restrictions, fetching tweets for the testimonials on the https://create.t3.gg landing page failed and displayed three dummy tweets instead.
In order to fix this, this PR:

- scraps the tweet fetch completely
- hard codes the five tweets that were initially listed to be fetched

Unfortunately this means that any data per tweet (engagement analytics, as well as the name and handle) is now static.

---

## Screenshots

The new carousel entries:

image

image

image

image

image

### CODE REVIEW

File: www/src/components/landingPage/tweets/tweets.astro [hunk 1]
```diff
@@ -1,65 +1,90 @@
...
-    // Feel free to add more tweets here
-    "1544909672137867264", // https://twitter.com/ajcwebdev/status/1544909672137867264
-    "1595652166173458434", // https://twitter.com/tomdoes_tech/status/1595652166173458434
-    "1553580714591158272", // https://twitter.com/synecdokey/status/1553580714591158272
-    "1553191258771841024", // https://twitter.com/royanger/status/1553191258771841024
-    "1570054715240763393", // https://twitter.com/jonhigger/status/1570054715240763393
-  ]);
-}
 const THREE_HOURS = 3 * 60 * 60;
 Astro.response.headers.set("Cache-Control", `public, max-age=${THREE_HOURS}`);
```
Comment (juliusmarminge) [inline www/src/components/landingPage/tweets/tweets.astro:89]: we can remove this cache now too

Comment (tbreslein) [reply www/src/components/landingPage/tweets/tweets.astro]: As I thought, thank you!

File: www/src/components/landingPage/tweets/tweets.astro [hunk 2]
```diff
@@ -1,67 +1,89 @@
...
 
-const token = import.meta.env.TWITTER_BEARER_TOKEN;
-const fetchTweets = async (ids: string[]) => {
-  const url =
-    "https://api.twitter.com/2/tweets" +
-    `?ids=${ids.join(",")}` +
-    "&tweet.fields=created_at,public_metrics" +
-    "&user.fields=name,profile_image_url,username,verified" +
-    "&expansions=author_id";
+let tweets: Tweet[] = [
```
Comment (juliusmarminge) [inline www/src/components/landingPage/tweets/tweets.astro:6]: i think this can be in a separate file along with the typedef or smth

Comment (tbreslein) [reply www/src/components/landingPage/tweets/tweets.astro]: Good point. I moved it to the typedef file, and renamed that file as well as const `featuredTweets`.

**Review by c-ehrlich — APPROVED** [review]
looks good. lets get the site working again, can fix any nits in a future pr

**Pull request merged**